src/components/admin/store/CollectionDetailsForm/index.tsx

```diff
@@ -0,0 +1,182 @@
+import EventDetailsFormItem from '@/components/admin/event/EventDetailsFormItem';
+import { Button } from '@/components/common';
+import { config, showToast } from '@/lib';
+import { StoreAPI } from '@/lib/api';
+import { MerchCollection } from '@/lib/types/apiRequests';
+import { PublicMerchCollection } from '@/lib/types/apiResponses';
+import { reportError } from '@/lib/utils';
+import Link from 'next/link';
+import { useRouter } from 'next/navigation';
+import { useState } from 'react';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { BsArrowRight } from 'react-icons/bs';
+import style from '../ItemDetailsForm/style.module.scss';
+
+type FormValues = MerchCollection;
+
+interface IProps {
+  mode: 'create' | 'edit';
+  defaultData?: Partial<PublicMerchCollection>;
+  token: string;
+}
+
+const CollectionDetailsForm = ({ mode, defaultData = {}, token }: IProps) => {
+  const router = useRouter();
+  const initialValues: FormValues = {
+    title: defaultData.title ?? '',
+    // ACM blue
+    themeColorHex: defaultData.themeColorHex ?? '#62b0ff',
```

Comment:
wondering if there's a way to pull the css var directly from a sass sheet for this

other option is just referencing a css variable if that works 
```suggestion
    themeColorHex: defaultData.themeColorHex ?? 'var(whatever-blue-variable)',
```